packages/openassistants/openassistants/contrib/advisor_function.py

```diff
@@ -28,6 +28,12 @@ async def advisor_query(
     BEARER_TOKEN = os.getenv("ADVISOR_BEARER_TOKEN")
     ADVISOR_API_BASE = os.getenv("ADVISOR_API_BASE")
 
+    if ADVISOR_API_BASE is None:
+        raise ValueError("ADVISOR_API_BASE environment variable not set")
+
+    if BEARER_TOKEN is None:
+        raise ValueError("ADVISOR_BEARER_TOKEN environment variable not set")
+
```

Comment:
I dont think these should be required- these will raise even if youre not using this file right? or no b/c python

Reply:
Will only raise if this function is called.

Reply:
I'm not crazy about config that can only be set with env variables, but... that's not new with this change.

Reply:
Also a bit surprised to see advisor specific things in openassistants, but I guess that's why it's in `contrib`